Sources/OpenAPIRuntime/Conversion/Converter+Common.swift

```diff
@@ -55,7 +55,7 @@ public extension Array where Element == HeaderField {
     }
 }
 
-public extension Converter {
+extension Converter {
```

Comment:
We were inconsistent in whether we put the access modifier on the extension or the methods, so I moved it to the methods. That also makes it easier to copy methods into `Deprecated.swift` when their time comes 😬